config/kafkaconnect/config.go

```diff
@@ -0,0 +1,57 @@
+package kafkaconnect
+
+import (
+	"time"
+
+	"github.com/crossplane/upjet/pkg/config"
+	"github.com/upbound/provider-aws/config/common"
+)
+
+// Configure adds configurations for the kafkaconnect group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_mskconnect_connector", func(r *config.Resource) {
+		// This will always refer to a Cluster in the kafka api group, if it refers to any managed resource at all,
+		// but which property from the status of that cluster to use depends on the authentication mechanism chosen.
+		delete(r.References, "kafka_cluster.apache_kafka_cluster.bootstrap_servers")
+		r.References["kafka_cluster.apache_kafka_cluster.vpc.security_groups"] = config.Reference{
+			TerraformName:     "aws_security_group",
+			SelectorFieldName: "SecurityGroupSelector",
+			RefFieldName:      "SecurityGroupRefs",
```

Comment:
Is there a particular reason why you use singular resource names when defining references here?

As I understand, more than one security group can be passed, so we may consider removing these definitions.

Reply:
This is the standard I've observed in most cases with a plural reference field. Just parsing the names from an English perspective, a "Security group selector" doesn't give much of an indication one way or the other as to whether there's a single security group or multiple supported. 

Security Group Refs is grammatically correct, indicating multiple references to one security group each, while the default "Security Groups Ref" would imply one reference to multiple security groups. 

It would be great to have consistency on these sorts of names, and I thought this was following the existing prevailing standard, but I haven't gone through and looked at them more broadly to see how many multi-references use which patterns.

Reply:
Looking through them more broadly wasn't too hard:
`git grep -A 4 'r.ResolveMultiple'`

In provider-upjet-aws, there are 232 multiple references, including the ones added in this PR. Only 36 contain the `sRefs` string in their target field name, from which I infer that the vast majority have been customized in this way, or have a non-plural terraform field name. 

I'd like to be able to make upjet smart enough to handle simple plurals, but in order for that to be a non-breaking change, we'd have to go manually configure those 36 examples to use the old style, and then all the other upjet-based providers would have to do the same in order to consume the new version. So it's probably not worth it. 

It would be great to be able to have a check for new field references, but maybe we can just enforce it manually in code review.

Reply:
Thank you for the detailed explanation, then it seems logical to continue in its current form.